components/layout_2020/table/construct.rs

```diff
@@ -825,8 +840,25 @@ where
                     ::std::mem::forget(box_slot);
                 },
                 DisplayLayoutInternal::TableCaption => {
-                    // TODO: Handle table captions.
-                    // We are doing this until we have actually set a Box for this `BoxSlot`.
```

Comment:
This comment is for the  `::std::mem::forget(box_slot);` line which is still present in this patch. Should we retain the comment? The other match arms still have it and it makes sense to comment on `mem::forget` calls since that is not seen in typical code that just lets the destructor run.